statsmodels/tsa/statespace/tests/test_varmax.py

```diff
@@ -49,6 +49,7 @@ def test_mle(self):
             self.model.enforce_invertibility = True
             assert_allclose(results.llf, self.results.llf, rtol=1e-5)
 
+    @pytest.mark.smoke
```

Comment:
smoke is a misnomer
this is not a "smoke" test, it just doesn't use verified numbers

maybe test_start_params or something like that would be a better name